lib/internal/process/pre_execution.js

```diff
@@ -160,6 +160,9 @@ function prepareExecution(options) {
 }
 
 function setupSymbolDisposePolyfill() {
+  if (!getOptionValue('--experimental-explicit-resource-management')) {
+    return;
+  }
```

Comment:
The fact that we have those symbols available allows for TypeScript to implement this correctly on their side. I don't think having these symbols in place cause any harm